### ISSUE TITLE
Classifying all notes in vault instead of just the active note?

### ISSUE THREAD

**Issue by JroachAu** [HyeonseoNam/auto-classifier#17]
Can we look into auto-classifying all notes in the vault with multiple relevant tags.

**Comment by Nathandomes** [comment]
I agree, would be very useful. I actually came to the Issues tab just to request this too.

I currently use another note taking app, and have a huge stash of uncategorized and untagged notes. Importing this into Obsidian would make for a bunch of loose (messy) notes. If I can have an auto-classifier go through all the back catalogue of notes and add tags, that would be amazing.

**Comment by mattjoyce** [comment]
Perhaps the way to approach this, is to have a folder.  Any file placed in the folder will be classified, and move to another folder.
I just feel automating changes for every file is likely to end up with some regret.

**Comment by creuzerm** [comment]
The python code I use to accomplish this goal first checks to make sure there are no tags already in place. Trying to find that balance between risking damaging existing human curated tags vs tagging 'empty' files.

**Comment by khallmark** [comment]
> The python code I use to accomplish this goal first checks to make sure there are no tags already in place. Trying to find that balance between risking damaging existing human curated tags vs tagging 'empty' files.

Care to share the script?
